src/processors/peersync/index.ts

```diff
@@ -21,6 +21,9 @@ const pollAllPeers = async (peerManager: PeerManager): Promise<void> => {
         await peerManager.poll(peer)
       }
     }
+
+    // induce a loop to its self after timeout has completed.
+    await setTimeout(pollAllPeers, ENV.PEERSYNC_POLL_INTERVAL_MS, peerManager)
```

Comment:
`setTimeout` does not return Promise. Therefore, this await may not be necessary.

Reply:
yup agreed